### ISSUE TITLE
[Request]: change car speed for other players

### ISSUE THREAD

**Issue by bob6456** [Yimura/YimMenu#2613]
### Solution

1

### Reason

it would be cool to make the cars from my friends/other players faster that they can drive 500kmh or change the gravity or something for other players cars becuase i think in stand is a option to change the horsepower for other players that they can go faster with car

i think that this would be a cool option that they can drive fast with cars or can you make a lua script that this works if yes can someone do this that i can change the speed for other players

**Comment by pierrelasse** [comment]
Yimura previously said that chaning a vehicle's handling for other players is impossible since it's handled completly on their end.
Tho maybe there is a way to still increase vehicle speed for other players. That would be cool to have.

**Comment by CypherMorgan** [comment]
> Yimura previously said that chaning a vehicle's handling for other players is impossible since it's handled completly on their end. Tho maybe there is a way to still increase vehicle speed for other players. That would be cool to have.

I've seen some P2C's being able to change vehicle's speed for other players, so maybe there's something changed with that.

**Comment by Yimura** [comment]
> Yimura previously said that chaning a vehicle's handling for other players is impossible since it's handled completly on their end. Tho maybe there is a way to still increase vehicle speed for other players. That would be cool to have.

Yeah 99% of handling is client-side controlled however there's this random native that can used to modify the speed of another player's vehicle.

**Comment by TheGreenBandit** [comment]
Just take control of the vehicle and set the speed or velocity lol

**Comment by Yimura** [comment]
> Just take control of the vehicle and set the speed or velocity lol, handling has nothing to do with the max speed or current speed

That’s a different thing and requires you keep taking control and doesn’t give any control to the user in the vehicle. The native in question that I’m talking about increases the speed of the vehicle while modifying it permanently for the other player.
https://natives.gta.menu/natives/0xAB54A438726D25D5
